code/go/0chain.net/smartcontract/dbs/event/blobber.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/guregu/null"
 )
 
+const ActiveBlobbersTimeLimit = 60 * 60 // 1 hour
```

Comment:
1 hour might be too long, blobbers are sending heart beat transactions every 60 second, so maybe 5 minutes is enough to say the blobber is offline.

Reply:
updated to 5  mins